misc/python/materialize/scratch.py

```diff
@@ -138,13 +136,43 @@ def launch(
     tags["git_ref"] = git.describe()
     tags["ami-user"] = ami_user
 
+    ec2 = boto3.client("ec2")
+    groups = ec2.describe_security_groups()
+    security_group_id = None
+    for group in groups["SecurityGroups"]:
+        if group["GroupName"] == security_group_name:
+            security_group_id = group["GroupId"]
+            break
+
+    if security_group_id is None:
+        vpcs = ec2.describe_vpcs()
+        vpc_id = None
+        for vpc in vpcs["Vpcs"]:
+            if vpc["IsDefault"] == True:
+                vpc_id = vpc["VpcId"]
+                break
+        if vpc_id is None:
+            vpc = ec2.create_default_vpc()
```

Comment:
Oh, hallelujah! I didn't realize you could recreate the default VPC after deleting it. This is so much easier.